pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -110,7 +110,11 @@ func fixBundleExtracted(bundlePath string, preset crcpreset.Preset) func() error
 					return err
 				}
 			}
-			bundlePath = constants.GetDefaultBundlePath(preset)
+		} else {
+			var err error
+			if bundlePath, err = bundle.Download(preset, bundlePath); err != nil {
+				return err
+			}
```

Comment:
A bit confused that we still have 
```
		if bundlePath == constants.GetDefaultBundlePath(preset) {
			logging.Infof("Downloading %s", constants.GetDefaultBundle(preset))
			// In case of OKD or podman bundle then pull the bundle image from quay
			// otherwise use mirror location to download the bundle.
			if preset == crcpreset.OKD || preset == crcpreset.Podman {
				if err := image.PullBundle(preset, ""); err != nil {
					return err
				}
			} else {
				if err := bundle.DownloadDefault(preset); err != nil {
					return err
				}
			}
```
on top of this, this looks redundant? (as in, could these 2 codepaths become a single one?)

Reply:
@cfergeau yes, in next iteration we should have same codepath. Just using `bundle.Download`